.github/workflows/dependency-check.yaml

```diff
@@ -16,6 +16,9 @@ jobs:
         with:
           java-version: '17'
           distribution: 'corretto'
+      - uses: dtolnay/rust-toolchain@1.79.0
+      - name: Install cargo cross
+        run: cargo install cross
```

Comment:
Can we leave this out for now, and skip Rust in the Maven build for this workflow? It's very time consuming to compile the Rust code. I'll raise a separate issue for adding Rust to the dependency checker.